### ISSUE TITLE
linter: eslint/no-fallthrough

### ISSUE THREAD

**Issue by highvoltz** [oxc-project/oxc#12376]
### What version of Oxlint are you using?

1.70

### What command did you run?

Using oxlint VS Code extension

### What does your `.oxlintrc.json` config file look like?

```jsonc
{
  "rules": {
    "no-async-promise-executor": "error",
    "no-empty": "warn",
    "no-fallthrough": "error",
    "no-global-assign": "off",
    "no-undef": "off",
    "no-unused-vars": "off",
    "sort-imports": "warn",
    "strict": "off"
  },
  "plugins": ["eslint", "oxc", "unicorn"],
  "ignorePatterns": [],
  "env":  {},
  "globals": {
    "QAggregateError": "readonly",
    "define": "readonly",
    "GeolocationPositionError": "readonly",
    "google": "readonly",
    "pdfMake": "readonly",
    "PublicKeyCredential": "readonly",
    "require": "readonly",
    "showDirectoryPicker": "readonly",
    "window": "readonly"
  }
}
```

### What happened?

It's wrongly considering switch cases below commented out cases as in violation of the no-fallthrough rule.
In the following example, the oxlint VS Code extension will mark the `case 3:` as in violation
```js
function test() {
	let number = 5
	switch (number) {
		case 1:
		// case 2:
		case 3:
		case 4:
			console.log("Number is 1, 3, or 4");
			break
	}
}
```

**Comment by lambda47** [comment]
It can be configured through {"allowEmptyCase": true}.
By default, this rule does not require a fallthrough comment after an empty case only if the empty case and the next case are on the same line or on consecutive lines.
